### PR TITLE
docs: Testing policies

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: docs/cli/policy/testing.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+*Note: At this time it is not possible to run only a single test, you have to run all tests in the entire directory
+
+
+`cloudquery policy test <selector/to/root/policy> <output/directory>`
+
+#### Example:
+``` bash
+cloudquery policy test ../aws test/snapshot-data
+```
+
```
[the remark this answers is not in the document]
Comment (bbernays) [reply docs/cli/policy/testing.md:62]: The output right now is super super super bad (I say this cause I wrote it). At this point we are documenting it for our own usage not because I think customers will be using it in the short/near feature.

File: docs/cli/policy/testing.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+### Running tests
+
+
+The test running utility takes a relative or absolute path to a policy and a path to a directory where test snapshots are stored. The path of the test matches the selector for a corresponding check in the policy. If the results of the check do not match the expected results the test run will end immediately
+
+*Note: At this time it is not possible to run only a single test, you have to run all tests in the entire directory
+
+
+`cloudquery policy test <selector/to/root/policy> <output/directory>`
```
Comment (bbernays) [inline docs/cli/policy/testing.md:56]: document the destructive nature of a restore (truncates data and destroys FKs) so should only be run on test/disposable database

File: docs/cli/policy/testing.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Testing Policies
+
+In order to continue to rely on policies we need to ensure that we are able to handle edge cases as well as any changes do not result in a regression. To do that the most common way is to have a suite of test cases and ensure the output matches the expected output for specific inputs
```
Comment (disq) [inline docs/cli/policy/testing.md:3]: This first sentence ends in a weird way. Maybe:
```suggestion
In order to continue to rely on policies we need to ensure that we are able to handle edge cases and make sure any changes do not result in a regression. To do that the most common way is to have a suite of test cases and ensure the output matches the expected output for specific inputs.
```

File: docs/cli/policy/testing.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+`cloudquery policy snapshot ../aws//foundational_security/apigateway/ApiGateway.2 test/snapshot-data`
+
+
+:::caution
+Prior to committing to remote repository:
+
+validate that there is no sensitive information (account IDs, credentials or PII) in the csv
+
+``` bash
+go run tools/find-sensitive-strings.go
```
[the remark this answers is not in the document]
Comment (bbernays) [reply docs/cli/policy/testing.md:43]: Great catch. The idea I had was for each policy repo to implement their own scripts under the same name

**Review by shimonp21 — APPROVED** [review]
